docs/DeveloperGuide.md

```diff
@@ -675,8 +693,39 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 ### **4. Enhanced Flexibility in Phone Number Parameter Input**
 
 ### **5. Find Feature Enhancement 1**
+**Potential Flaw in Current Implementation**<br>
+Currently, LinkMeIn only allows searching through the list of companies by the `COMPANY_NAME` parameter. However, 
+users might want to search through the list using other parameters, like `RECRUITER_NAME`, `PRIORITY` and `ROLE`.
+
+**Proposed Enhancement**<br>
+We plan to expand the current find command’s capability to allow for search using other parameters. The users will 
+be able to specify the prefix that corresponds to the parameter they wish to use for the search, before the keyword(s). 
+The prefixes used will be consistent with the rest of the application, in regard to what parameter they represent.
+
+Here are the new suggested formats :
+* Find using `RECRUITER_NAME` : `find n/KEYWORD`
+* Find using `PRIORITY`: `find pr/KEYWORD`
+* Find using `Role`: `find r/KEYWORD`
+
+**Examples**<br>
+* `find n/John Doe`
+* `find pr/High`
+* `find r/Software Engineer`
 
 ### **6. Find Feature Enhancement 2**
```

Comment:
**Enhance Find Feature to Search for Exact Company Names Only**

will this be a better header?

Reply:
Changed to this: Enhance Find Feature to Allow for Search of Exact Company Names. Is this ok?